lib/authenticators/okta/api.go

```diff
@@ -64,8 +64,21 @@ func (pa *PasswordAuthenticator) ValidateUserPush(username string) (PushResponse
 	return pa.validateUserPush(username)
 }
 
+// New creates a new public authenticator, but poiting to an explicit authenticator url
```

Comment:
s/poiting/pointing/

Also, please move this up in the file to before `NewPublic`.

Reply:
fixed